src/dou_dag_generator.py

```diff
@@ -413,11 +455,7 @@ def create_dag(self, specs: DAGConfig, config_file: str) -> DAG:
 
             with TaskGroup(group_id="exec_searchs") as tg_exec_searchs:
 
-                # is it a single search or a list of searchers?
-                if isinstance(specs.search, list):
-                    searches = specs.search
-                else:
-                    searches = [specs.search]
+                searches = self._ensure_list_of_searches(specs=specs)
```

Comment:
```suggestion
                searches = specs.search
```

Já é lista (vide comentário acima).